packages/react/src/response.ts

```diff
@@ -0,0 +1,38 @@
+import { ActionEventDataState, JSONValue } from './template'
+
+export const res = (payload: JSONValue): ServerResponseDataState => {
```

Comment:
usually I would do `export function` here

Reply:
Also I think we should have a more descriptive name. maybe `createResponse`?

Reply:
Sounds good!

Reply:
const vs let tho